cloudwatch-logs-s3-forwarder/README.md

```diff
@@ -0,0 +1,11 @@
+# Cloud Watch Log group subscriptions to S3 forwarder
+
+This module lets you forward logs from Cloud Watch Logs to an S3 bucket. 
+
+The module creates Cloud Watch subscriptions for one or more Log groups, and targets the subscription to a Lambda function that is provisioned by this module. The Lambda function base64 deocde and decompress the Cloud Watch payload and moves it to a S3 bucket.
+
+ 
+You can use filter expressions to copy only a subset of the log entries. . 
+
+## Prerequisites  
+You need to build and upload the lambda artifact to an S3 bucket  under your control before you apply this module.  [The Lambda artifact can be found here](https://github.com/TeliaSoneraNorge/cloudwatch-subscriptions-s3-writer)
```

Comment:
This is a private repo, I'm assume we should move this to a oss repo?

Reply:
If/when moving it, I think it's worth considering to add the source code to the module or the other way around (mentioned this above).

Reply:
The concourse pipeline that compiles and assembles the lambda is finished. It would be easy to make the artifact bucket public if we want.

Reply:
I think this is a broader topic (we really need a public bucket where we can publish thinks like this). This could however be added as an issue to this repo and be done in a separate PR so we can get this merged without feature creeping 👍

Reply:
@itsdalmo I'll also add a ticket to merge terraform + Java repos, so we can get it merged.